google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorkerPool.java

```diff
@@ -192,7 +193,8 @@ public abstract static class Builder {
   /** Static setting for connection pool. */
   private static Settings settings = Settings.builder().build();
 
-  public ConnectionWorkerPool(
+  @VisibleForTesting
```

Comment:
VisibleForTesting is for test only I think

Reply:
Yes. Other users will use this ctor? I thought they only need to use the settings on the pool?

Reply:
Done.